src/plugins/api/qmlpluginapi.cpp

```diff
@@ -118,14 +120,83 @@ QQmlListProperty<Score> PluginAPI::scores()
 bool PluginAPI::writeScore(Score* s, const QString& name, const QString& ext)
```

Comment:
I think we can use exportProjectScenario here

exportProjectScenario()->exportScores({currentNotation}, outPath);

Reply:
I have changed the PR to use exportProjectScenario. It indeed simplifies things but I still have to create a notation writer to figure out the most suitable unit type for export, and I had to tweak the exportProjectScenario's code to get it working with the MSCZ writer (it supports only `MULTI_PART` unit type but does not support the `writeList()` method, I have changed the export scenario's code to use a usual `write()` method in this case if exporting only one score).